tests/serverauth-crd.yaml

```diff
@@ -0,0 +1,311 @@
+                      type: object
+                    type: array
+                  unauthenticated:
+                    description: Authorizes unauthenticated clients to access a server.
+                    type: boolean
+                type: object
+              server:
+                description: |-
+                  Identifies servers in the same namespace for which this authorization applies.
+                  Only one of `name` or `selector` may be specified.
+                oneOf:
+                - required:
+                  - name
+                - required:
+                  - selector
+                properties:
+                  name:
+                    description: References a `Server` instance by name
+                    pattern: ^[a-z0-9]([-a-z0-9]*[a-z0-9])?$
+                    type: string
+                  selector:
+                    description: A label query over servers on which this authorization
+                      applies.
+                    oneOf:
+                    - required:
+                      - matchLabels
+                    - required:
+                      - matchExpressions
+                    properties:
+                      matchExpressions:
+                        items:
+                          properties:
+                            key:
+                              type: string
+                            operator:
+                              enum:
+                              - In
+                              - NotIn
+                              - Exists
+                              - DoesNotExist
+                              type: string
+                            values:
+                              items:
+                                type: string
+                              type: array
+                          required:
+                          - key
+                          - operator
+                          type: object
+                        type: array
+                      matchLabels:
+                        type: object
+                        x-kubernetes-preserve-unknown-fields: true
+                    type: object
+                type: object
+            required:
+            - server
+            - client
+            type: object
+        required:
+        - spec
+        type: object
+    served: true
+    storage: true
+status:
+  acceptedNames:
+    kind: ServerAuthorization
+    listKind: ServerAuthorizationList
+    plural: serverauthorizations
+    shortNames:
+    - saz
+    singular: serverauthorization
+  conditions:
+  - lastTransitionTime: "2021-12-13T21:51:27Z"
+    message: no conflicts found
+    reason: NoConflicts
+    status: "True"
+    type: NamesAccepted
+  - lastTransitionTime: "2021-12-13T21:51:27Z"
+    message: the initial names have been accepted
+    reason: InitialNamesAccepted
+    status: "True"
+    type: Established
+  storedVersions:
+  - v1beta1
```

Comment:
this status part should probably be removed, not part of the original crd

Reply:
done